application-forum-ui/src/main/resources/ForumCode/AnswerSheet.xml

```diff
@@ -53,6 +53,26 @@
         &lt;/dl&gt; 
       &lt;/div&gt;
     {{/html}}
+  #elseif ($request.commentEditForm == 'true')
+    {{html clean="false"}}
+      ## XWIKI-17318
```

Comment:
What does this JIRA issue number mean? You need to be more explicit. The very least is to write down the issue title, but it would be good to also mention what is the relation between the specified JIRA issue and the following lines of code.